scripts/action-recognition/test_recognizer.py

```diff
@@ -153,11 +153,11 @@ def test(ctx, val_data, opt, net):
     for i, batch in enumerate(val_data):
         data, label = batch_fn(batch, ctx)
         outputs = []
-        for X in data:
+        for _, X in enumerate(data):
+            X = X.reshape((-1,) + X.shape[2:])
```

Comment:
Yes, you are correct. The output of dataloader now has 5 dimensions. And for our 3D models, the output of dataloader will be 6 dimensions. (N, num_segments, 3, video_length, input_size, intput_size). Let's talk offline why it will cause crash. Thank you.